tnefparse/tnef.py

```diff
@@ -399,25 +396,20 @@ def to_zip(data, default_name='no-name', deflate=True):
     tozip = {}
     for attachment in tnef.attachments:
         filename = attachment.name or default_name
-        L = len(tozip.get(filename, []))
-        if L > 0:
+        if tozip.get(filename):
+            length = len(tozip.get(filename))
             # uniqify this file name by adding -<num> before the extension
-            root, ext = os.path.splitext(filename)
-            tozip[filename].append((attachment.data, str("%s-%d%s" % (root, L + 1, ext))))
-        else:
-            tozip[filename] = [(attachment.data, filename)]
+            filename = filename.with_stem(f"{filename.stem}-{length + 1}")
```

Comment:
There is no coverage for this line. It would be great to have a test for it.

Reply:
This will be done in #89